classes/ActionScheduler_ListTable.php

```diff
@@ -0,0 +1,354 @@
+<?php
+
+/**
+ * Implements the admin view of the actions.
+ */
+class ActionScheduler_ListTable extends PP_List_Table {
+	/**
+	 * The package name. It is used also as the domain for the translations
+	 */
+	protected $package = 'action-scheduler';
+
+	/**
+	 * Columns to show (name => label). The label is automatically
+	 * translated before rendering
+	 */
+	protected $columns = array(
+		'hook'   => 'Hook',
+		'status' => 'Status',
+		'args'   => 'Arguments',
+		'group'  => 'Group',
+		'recurrence' => 'Recurrence',
+		'scheduled'  => 'Scheduled Date',
+		'claim_id'   => 'Claim ID',
+		'comments'   => 'Log',
+	);
+
+	protected $row_actions = array(
+		'hook' => array(
+			'run' => array( 'Run', 'Process the action now as if it were run as part of a queue' ),
```

Comment:
These strings should be made available for translation and passed through i18n functions.